airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -12,9 +12,9 @@ java {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.40.10'
+    cdkVersionRequired = '0.41.1'
     features = ['db-sources', 'datastore-postgres']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
Was this intentional or done for testing?

Reply:
Forgot to reply - sorry
This is the usual way of operation when making changes in CDK.
Until that CDK is published as a new version, the connector needs to consume a locally built CDK.
Once you publish a CDK we change this back to false ahead of merging.